corehq/apps/analytics/tasks.py

```diff
@@ -38,6 +38,19 @@
 HUBSPOT_EXISTING_USER_INVITE_FORM = "7533717e-3095-4072-85ff-96b139bcb147"
 HUBSPOT_COOKIE = 'hubspotutk'
 
+ANALYTICS_RETRIES = 3
+ANALYTICS_SLEEP = 5
+
+def _persistent_analytics_post(func, retries=ANALYTICS_RETRIES, sleep=ANALYTICS_SLEEP):
```

Comment:
i pass closures to the retry function, defined in each analytics task. if there is a better way to have each of those function execute in a try except, i would be happy to change it but this seemed cleaner than decorating private internal functions